### DISCUSSION TITLE
System requirements?

### DISCUSSION THREAD

**Discussion by mccarthy-m-g** [posit-dev/positron#3670] [Q&A] [answered]
Would it be possible to add system requirements to the wiki/FAQ?

The wiki states that positron is available for Mac/Windows/Linux, but doesn’t state a minimum required version.

Thanks!

**Comment by juliasilge** [comment, marked as answer]
Thanks for the feedback! We'll get that added, but in the meantime, you can expect Positron to work for the same OS versions [as VS Code](https://code.visualstudio.com/download).

**Reply by olivroy** [reply]
I am on macOS 10.15. Positron works, but can't find my R version. It can find my Python installation though.

**Reply by juliasilge** [reply]
@olivroy Can you open a new discussion so we can get some details from you?
